src/System.Device.Gpio/System/Device/I2c/I2cController.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System.Collections.Generic;
+
+namespace System.Device.I2c
+{
+    public sealed partial class I2cController : II2cController
+    {
+        private readonly Dictionary<(int, int), I2cDevice> _openDevices;
```

Comment:
this probably should be ConcurrentDictionary or locked everywhere

Reply:
Added change for this.

Reply:
Not included any longer.